docs/tools/vdb_table/data/Hyperspace.json

```diff
@@ -0,0 +1,180 @@
+{
+  "name": "Hyperspace",
+  "links": {
+    "docs": "https://docs.hyper-space.io/hyperspace-docs",
+    "github": "https://github.com/hyper-space-io/",
+    "website": "https://www.hyper-space.io/",
+    "poc_github": "https://github.com/YanivHyper-Space",
+    "slug": "hyper-space"
+  },
+  "oss": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "license": {
+    "value": "Hyper-space-2.0",
+    "source_url": "",
+    "comment": ""
+  },
+  "dev_languages": {
+    "value": [
+      "python",
+      "java"
+    ],
+    "source_url": "",
+    "comment": ""
+  },
+  "vector_launch_year": 2023,
+  "metadata_filter": {
+    "support": "full",
+    "source_url": "",
+    "comment": ""
+  },
+  "hybrid_search": {
+    "support": "full",
+    "source_url": "https://docs.hyper-space.io/hyperspace-docs/reference/hyperspace-hybrid-search#hybrid-search",
+    "comment": "Support complex logic, including mutliplication and division of vector and keyword based scores. No limitations on numer of keyword fields or vector fields"
+  },
+  "facets": {
+    "support": "partial",
+    "source_url": "https://docs.hyper-space.io/hyperspace-docs/flows/data-collections/python-query-interface/aggregations",
+    "comment": ""
+  },
+  "geo_search": {
+    "support": "full",
+    "source_url": "https://docs.hyper-space.io/hyperspace-docs/flows/es-dsl-query-interface/candidate-filtering#match-between-geo-coordinates",
+    "comment": ""
+  },
+  "multi_vec": {
+    "support": "full",
+    "source_url": "https://docs.hyper-space.io/hyperspace-docs/flows/setting-up/building-and-running-queries/building-a-vector-search-query",
+    "comment": "can support large number of vectors in same query"
+  },
+  "sparse_vectors": {
+    "support": "",
+    "source_url": "",
+    "comment": ""
+  },
+  "bm25": {
+    "support": "full",
```

Comment:
please change this to "none", as this column specifically refers to BM25, which is a more complex formula than TF-IDF, which is what is implemented